python/common/org/python/types/Bool.java

```diff
@@ -27,6 +27,11 @@ public int hashCode() {
         return new java.lang.Boolean(this.value).hashCode();
     }
 
+    public org.python.types.Int __hash__() {
+        if (this.value == true) return org.python.types.Int.getInt(1);
+        else return org.python.types.Int.getInt(0);
```

Comment:
I'm surprised this isn't picked up by the code linter... but this formatting needs to be cleaned up.

Firstly, the body of if/else clauses should always be indented. Secondly "naked" if clauses shouldn't be used.

This code should look like:
```
if (this.value == true) {
    return org.python.types.Int.getInt(1);
} else {
    return org.python.types.Int.getInt(0);
}
```